lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,16 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    # horrible terrible temporary implementation
+    # room would eventually do all of this
+    ctxt = %Yggdrasil.Room.Context{ player: user_id }
```

Comment:
Yeah I should point out the comment above. Also this was originally done here because I was thinking about the player process crashing. I think this whole function will change completely. We just simply have nothing else.

I think at this point I'm just going to try and bump up the test coverage for things I think will stay and ask for this to be merged in.